pandas/io/sql.py

```diff
@@ -44,7 +44,7 @@ class DatabaseError(IOError):
 def _validate_flavor_parameter(flavor):
     """
     Checks whether a database 'flavor' was specified.
-    If not None, produces FutureWarning if 'sqlite' and
+    If not None, produces FutureWarning if 'SQLite' and
```

Comment:
I think the previous version with lowercase 'sqlite' is correct, because this is referencing the `flavor` parameter that's being checked, and it's looking for all lowercase (see line 51).

Reply:
You are definitely correct. SQLite is the proper capitalization for the library when used as a noun, but it looks like it needs to be lower case when being used. Thanks for spotting that.

Reply:
Should be good now I believe.